Controls/CkEditorMinimal/src/DotVVM.Contrib/Scripts/DotVVM.Contrib.CkEditorMinimal.js

```diff
@@ -0,0 +1,23 @@
+﻿
+function setupCkEditor(id) {
+    var editor = CKEDITOR.instances[id];
+    if (editor) {
+        editor.destroy(true);
+    }
+    CKEDITOR.replace(id);
+    CKEDITOR.add
```

Comment:
I'm pretty sure that this statement does nothing, did you want to invoke the function?